### PR TITLE
Fixes actual price vs price validation check

### PR DESCRIPTION
**Pull request by rehan99000**
**Description:**
Fixes actual price vs retail price validation check.

### CODE REVIEW

File: ecommerce/subscriptions/api/v2/serializers.py [hunk 1]
```diff
@@ -184,8 +184,8 @@ def validate(self, subscription):
         Validate subscription attributes corresponding to selected subscription_type.
         """
         subscription_type = subscription.get('subscription_type')
-        subscription_price = subscription.get('subscription_price')
-        subscription_actual_price = subscription.get('subscription_actual_price')
+        subscription_price = int(subscription.get('subscription_price'))
```
Comment (zubair-arbi) [inline ecommerce/subscriptions/api/v2/serializers.py:187]: Shouldn't this value be a float?
CC: @ziafazal

**Review by zubair-arbi — CHANGES_REQUESTED** [review]
I have one question.

**Review by burhandodhy — APPROVED** [review]

File: ecommerce/subscriptions/api/v2/serializers.py (hunk 1 above)
Comment (rehan99000) [reply ecommerce/subscriptions/api/v2/serializers.py]: done

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Pull request merged**